doc/src/manual/embedding.md

```diff
@@ -257,8 +257,8 @@ out from under you, rendering pointers invalid.
 The GC can only run when Julia objects are allocated. Calls like `jl_box_float64` perform allocation,
 and allocation might also happen at any point in running Julia code. However, it is generally
 safe to use pointers in between `jl_...` calls. But in order to make sure that values can survive
-`jl_...` calls, we have to tell Julia that we hold a reference to a Julia value. This can be done
-using the `JL_GC_PUSH` macros:
+`jl_...` calls, we have to tell Julia that we hold a reference to a Julia value, a root, this procedure
+is called GC rooting. This can be done using the `JL_GC_PUSH` macros:
```

Comment:
I would suggest stating the "root" term only once, to make it a little smoother to read, then expanding on what a root means a little bit more.  Something like: `...we have to tell Julia that we still hold a reference to a Julia value, a process called "GC rooting".  Rooting a value will ensure that the garbage collector does not accidentally identify this value as unused and free the memory backing that value.  This can be done...`

Reply:
Might also be worth mentioning the origin of the term "root": they are roots of the graph of objects which the GC will traverse; cf. roots of trees in the graph theory sense. (At least, I've always assumed so and I went looking for the origin of the term. I couldn't find it, but it was no later than 1969: https://www.cs.purdue.edu/homes/hosking/690M/p611-fenichel.pdf)

Reply:
Hopefully the slight modified text is ok. I found "root" in the linked text; strictly not "[GC] rooting", so better with way to include both?